### PR TITLE
Add blog post 'Revamping Cloud and Monitor Pages'

### PR DESCRIPTION
**Pull request by dmarcoux**
Review: https://deploy-preview-227--openbuildservice.netlify.com/2019/06/06/revamping-cloud-ui/

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 68ceec1974979f6ff51648993f4aadc7b492bd37

https://deploy-preview-227--openbuildservice.netlify.com

File: _includes/partials/_revamping-ui-releases-history.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ We started off with the [Package pages](/2018/10/05/revamping-ui/) in October 20
 moved on to the [Project, User and Group pages](/2018/12/10/revamping-project-ui) in December 2018,
 continued with the [Request pages](/2019/02/15/revamping-request-ui/) in February 2019
 and migrated the [Configuration pages](/2019/03/03/revamping-configuration-ui/) in March 2019.
-We then finished the [Maintenance pages](/2019/04/30/revamping-maintenance-ui/) in April 2019
-and now, we focused on the [Search and Kiwi Editor pages](/2019/05/27/revamping-search-ui/)._
+We then finished the [Maintenance pages](/2019/04/30/revamping-maintenance-ui/) in April 2019,
+the [Search and Kiwi Editor pages](/2019/05/27/revamping-search-ui/) were completed in May 2019
+and now, we focused on the [Cloud and Monitor pages](/2019/06/06/revamping-search-ui/)._
```
Comment (eduardoj) [inline _includes/partials/_revamping-ui-releases-history.md]: s/revamping-search-ui/revamping-cloud-ui/

Comment (dmarcoux) [reply _includes/partials/_revamping-ui-releases-history.md]: Oops, good catch!

File: _posts/development/2019-06-06-revamping-cloud-ui.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: The Cloud and Monitor Pages Are Now Part of the Revamped User Interface
```
Comment (hennevogel) [inline _posts/development/2019-06-06-revamping-cloud-ui.md]: :thought_balloon:  ... *yawn* ...

**Comment by dmarcoux** [comment]
I reworked the title, the introduction and added a `What Is Next?` section. Let me know what you think.

**Comment by dmarcoux** [comment]
[Communicate the deliverable](https://trello.com/c/pkQNANBM/1167-communicate-the-deliverable)

**Comment by dmarcoux** [comment]
[Communicate the deliverable](https://trello.com/c/xXJwMs7k/1168-communicate-the-deliverable)

**Pull request merged**